### DISCUSSION TITLE
Language pack Pending application creation

### DISCUSSION THREAD

**Discussion by bakeromso** [AnySoftKeyboard/AnySoftKeyboard#2616] [Q&A] [answered]
Hi All,

Can anybody tell me what does the following message mean?
 Dutch: Source (not published reason: 'Pending application creation.') 

It says so [here](https://github.com/AnySoftKeyboard/AnySoftKeyboard/blob/master/addons/languages/PACKS.md).

Maybe I don't understand the way new language packs are contributed. In this case, the source for the Dutch is already made available, so are the devs waiting for someone to publish it? Or are they planning to publish it?

Anyways I could also publish that APK to the App Store myself if that helps.

**Comment by menny** [comment, marked as answer]
We are waiting for me 😃
I need to create the app and publish the first version. After that, any new commit to the packs will be?automatically published to the Play Store.

If you wish, you can take ownership of that pack, but we will not get automatic publishing in that case.

We also do not have any way to automatically publish to FDroid

**Comment by bakeromso** [comment]
Then I think it will be better for you to publish it if you would still like to do that. If not, then please let me know

**Reply by menny** [reply]
I published everything.
